src/Validation/JpValidation.php

```diff
@@ -25,13 +25,46 @@ class JpValidation extends LocalizedValidation
 {
     /**
      * Checks a phone number for Japan.
+     * Accepts the following format.
+     * - 0311112222
+     * - 03 1111 2222 or 09876 1 2222
+     * - 03-1111-2222 or 09876-1-2222
+     * - +81311112222
+     * - +81 3 1111 2222
+     * - +81-3-1111-2222
+     * - 09011112222
+     * - 090 1111 2222
+     * - 090-1111-2222
+     * - +819011112222
+     * - +81 90 1111 2222
+     * - +81-90-1111-2222
+     * - 0120111222
+     * - 0120 111 222
+     * - 0120-111-222
```

Comment:
Regarding digit number, I referred here.
http://www.soumu.go.jp/main_sosiki/joho_tsusin/top/tel_number/q_and_a.html

Should I add it?